README.md

```diff
@@ -470,6 +470,16 @@ Try updating the parser that you suspect has changed (`:TSUpdate {language}`) or
 If the error persists after updating all parsers,
 please [open an issue](https://github.com/nvim-treesitter/nvim-treesitter/issues/new/choose).
 
+#### I get `query error: invalid node type at position`
+
+This could be due a query file outside this plugin using outdated nodes,
+or due to an outdated parser.
+
+- Make sure you have the parsers up to date with `:TSUpdate`
+- Make sure you don't have more than one `parser` runtime directory.
+  You can execute this command `:echo nvim_get_runtime_file('parser', v:true)` to find all runtime directories.
+  If you get more than one path, remove the duplicated one.
```

Comment:
remove the one outside the `nvim-treesitter` directory, so nvim-treesitter can update it?

Reply:
I have expanded this.

> nvim-treesitter can update it

I think the parser is already up-to-date after running `TSUpdate`, it's just the other outdated parser taking precedence.